### ISSUE TITLE
How to make ALT+hjkl work on macOS?

### ISSUE THREAD

**Issue by dahmc** [zellij-org/zellij#1936]
I can navigate my splits with ALT+arrow keys, but not with ALT+hjkl.

Alt+hjkl always moves the cursor on my terminal input left/right or the history up/down.

I tried using bindkey -r on the sequences, but it still won't work. 

Any experience on how to use that on zsh with macOS?

**Comment by naosense** [comment]
Alt + hjkl works fine for me. Do you use iterm2?

**Comment by dahmc** [comment]
I'm on kitty, but I have the same issue with the macOS terminal

**Comment by naosense** [comment]
`Alt+nhjkl` did not work for me before, but i modified setting below(Normal -> Esc+) and it was fine. You can check if kitty has the relevant configuration.
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/2225426/201847321-1a75c8fd-4c25-4d89-be52-8105fd331f49.png">

**Comment by dahmc** [comment]
Thanks for the screenshot. 

I installed iTerm and did the same settings.

Alt-n works after changing, but hjkl does not, it still only operates on the line in the terminal.

**Comment by naosense** [comment]
Sry to hear that, are there some system-level shortcuts that override zj's shortcuts?

**Comment by dahmc** [comment]
Not that I know of... even when I run it without zsh config, or in bash, it's the same. hjkl will do cursor left/right or history up/down in the terminal.

This seems like a default behaviour that maybe I have to turn off?

**Comment by dahmc** [comment]
I just tested with a fresh installation of macOS and iTerm, and there it works. Gotta figure out what is interfering on my main system now

**Comment by naosense** [comment]
Glad it's working for you. Please let us know if you find out the cause.发自我的手机

**Comment by dahmc** [comment]
Ok I finally figured it out. Turns out I had https://github.com/dbalatero/VimMode.spoon running that enabled global vim hotkeys. 

Sorry for the confusion.

**Comment by Joxtacy** [comment]
> I am going to try on my moms macbook pro which is very close to a fresh install and post an update.

@pitoniak32 Did you ever get the chance to try it out on your mom's MacBook Pro?

**Comment by pitoniak32** [comment]
@Joxtacy I did not, magically these keys started working for me on my work computer, I'm not sure what happened 😅

**Comment by sherubthakur** [comment]
Ran into this issue too. Adding how I fixed this for anyone who tries to search for it.

For kitty this can be resolved using the instructions provided in https://sw.kovidgoyal.net/kitty/conf/#opt-kitty.macos_option_as_alt

tldr;
`macos_option_as_alt` option needs to be `left` (assuming you want the alt behavior from the left option key and don't care about the unicode input)
